ecommerce-ui/src/components/ShoppingCart.js

```diff
@@ -0,0 +1,29 @@
+import classes from "./ShoppingCart.module.css"
+import VacationRental from "./VacationRental"
+import Button from "../ui/Button"
+import PropTypes from "prop-types"
+
+function ShoppingCart({bnbCart, manageCart, closeCart}) {
+  let total = 0
+  let resultShoppingCartItems = bnbCart.map(item => <VacationRental key={item.id} bnb={item} manageCart={manageCart} action="Remove" />)
+  for(let i of bnbCart) {
```

Comment:
Nice. You could also handle this in the map above since you're already looping through them.